Makefile

```diff
@@ -61,15 +63,16 @@ IMAGE_EXTRA_TAG_NAMES ?=
 IMAGE_REPO := $(IMAGE_REGISTRY)/$(IMAGE_NAME)
 IMAGE_TAG := $(IMAGE_REPO):$(IMAGE_TAG_NAME)
 IMAGE_EXTRA_TAGS := $(foreach tag,$(IMAGE_EXTRA_TAG_NAMES),$(IMAGE_REPO):$(tag))
+CONTAINER_RUN_CMD ?= docker run -u "`id -u`:`id -g`"
```

Comment:
This should be specified before `SITE_BUILD_CMD`

Reply:
done ✅